TestCoin.ts

```diff
@@ -0,0 +1,24 @@
+import { Coin } from '../types';
+import { randomBytes } from 'crypto';
+
+export class TestCoin {
+  /**
+   * Creates a single test coin with given parameters
+   */
+  static create(params: Partial<Coin> = {}): Coin {
+    return {
+      id: params.id || `0x${randomBytes(32).toString('hex')}`,
+      owner: params.owner || `0x${randomBytes(32).toString('hex')}`,
+      amount: params.amount || BigInt(1000000),
```

Comment:
```suggestion
      amount: params.amount || bn(1000000),
```

Amount is a [BN](https://github.com/FuelLabs/fuels-ts/blob/master/packages/account/src/providers/coin.ts#L10)